### PR TITLE
Snyk to monitor vulnerabilities

### PR DESCRIPTION
**Pull request by easternbloc**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,15 +1,12 @@
...
-  - "4.2.2"
+  - "4.4.2"
 
 services:
   - redis-server
 
-cache:
-  directories:
-    - node_modules
-
```
Comment (daniel-ac-martin) [inline .travis.yml]: Have you seen what we are doing in lev-web? - https://github.com/UKHomeOffice/lev-web/blob/master/.travis.yml

Basically, we are caching but blowing away just node-sass as this is the thing that is locked to the node version.

Why did you make the change to stop caching anyway? (You're only testing against one node version anyway.)

Comment (daniel-ac-martin) [inline .travis.yml]: Or is it required for Snyk to even work?

**Comment by JoeChapman** [comment]
@easternbloc what do you want to do with this?

**Comment by easternbloc** [comment]
leaving it there as a reminder. I'm actually gonna be working on snyk integration tomorrow.

**Pull request closed without merging**